### PR TITLE
Add in Verkle Proof Benchmark

### PR DESCRIPTION
**Pull request by ControlCplusControlV**
I added in a quick benchmark for Verkle proofs, I was also getting benchmark times of ~2 hours and occasional hangs on 10,000,000 items (M1 Pro 32 GB of RAM), so I reduced that to make it more manageable while iterating on the library. I'm not sure how to write an edit benchmark or what you meant by that (inserting over values or stem updating?) so I was hoping you would be able to tackle that

### CODE REVIEW

**Comment by ControlCplusControlV** [comment]
squash and merge as always ofc

File: verkle-trie/Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ ark-ff = { version = "^0.3.0", default-features = false }
 ark-ec = { version = "^0.3.0", default-features = false }
 ark-serialize = { version = "^0.3.0", default-features = false }
 verkle-db = { path = "../verkle-db", default-features = false }
-
+tokio = { version = "1.21.1", features = ["full"] }
```
Comment (kevaundray) [inline verkle-trie/Cargo.toml:18]: why is this needed?

Comment (ControlCplusControlV) [reply verkle-trie/Cargo.toml:18]: It isn't will remove. I was toying around with async so it slipped in
